sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -260,7 +254,10 @@ def create_table(
         try:
             result = self._client.table.create(table_properties, **kwargs)
         except HttpResponseError as error:
-            _process_table_error(error, table_name=self.table_name)
+            try:
+                _process_table_error(error, table_name=self.table_name)
+            except AzureError as decoded_error:
```

Comment:
This should be `except HttpResponseError`?

Reply:
Yes, changed to `HttpResponseError`.